tests_and_analysis/test/euphonic_test/test_calculate_structure_factor.py

```diff
@@ -79,9 +79,9 @@ def test_calculate_structure_factor(self, material, qpt_ph_modes,
             sf = qpt_ph_modes.calculate_structure_factor()
         sf_file = os.path.join(get_sf_dir(material), expected_sf_file)
         expected_sf = sf.from_json_file(sf_file)
-        check_structure_factor(sf, expected_sf, sf_atol=8e-14, sf_rtol=3e-4,
+        check_structure_factor(sf, expected_sf, sf_atol=4e-4, sf_rtol=3e-4,
                                freq_atol=1e-4, freq_rtol=2e-5,
-                               freq_gamma_atol=0.5, sf_gamma_atol=1e-7)
+                               freq_gamma_atol=0.5, sf_gamma_atol=6e2)
```

Comment:
Appreciating that this is test rather than production code, but this makes me a bit nervous of raw numbers in what I guess is a unit conversion process. As we change the unit system, is this a permanent once-for-all change or should we  be providing unit-free code into which unit-ed numbers can be put?

Reply:
Yes, this could be a problem... I don't have time to fix it now but I've made an issue for it https://github.com/pace-neutrons/Euphonic/issues/175